site/src/components/steps.module.less

```diff
@@ -0,0 +1,19 @@
+.ant-steps-horizontal {
```

Comment:
If you want to customize Antd's Steps component (`@steps-description-max-width: 105px;`), I'd recommend doing it here rather than in the `antd-overrides.less`.